models/migrations/v91.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"code.gitea.io/gitea/models"
+
+	"github.com/go-xorm/xorm"
+)
+
+func removeLingeringIndexStatus(x *xorm.Engine) error {
+
+	var orphaned []*models.RepoIndexerStatus
```

Comment:
It is better to use type copied into this function otherwise it could start to fail if columns are changed in the future

Reply:
I.e. copy 

https://github.com/go-gitea/gitea/blob/5b902e2368fdb67eda4ba033f7167fadf72e4350/models/repo_indexer.go#L23-L27

above this function definition.

Reply:
Done